runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,4 +1,4 @@
-name: QuteServerPages
+name: Qute Web
 description: Automatically serves qute templates via HTTP.
```

Comment:
```suggestion
description: Qute Web serves Qute templates over HTTP, easily pairing with RESTEasy (Reactive or Classic) for more precise control.
```

Reply:
The _"easily pairing with RESTEasy (Reactive or Classic) for more precise control"_ part is not yet true so I was waiting for the follow-up PR to update the docs...